src/d8/aquifer.default.json

```diff
@@ -23,13 +23,13 @@
       "root": "files"
     }
   },
-  "extensions": {},
-  "refreshCommands": [
-    [
-      "updb"
-    ],
-    [
-      "cr"
-    ]
-  ]
+  "run": {
```

Comment:
Is the "scripts" property supposed to be a subproperty of "run"? In your documentation you specify adding commands to "scripts", which seems to work.